### PR TITLE
Add q3 roadmap

### PR DESCRIPTION
**Pull request by curquiza**

### CODE REVIEW

File: roadmaps/2021.md [hunk 1]
```diff
@@ -23,9 +23,37 @@ Nothing is definitive: the roadmap can be adapted depending on the need of the u
...
+- [ ] Cron job CI to run some SDKs tests
+- [ ] MeiliSearch (e.g. Transplant) documentation
+- [ ] Indexation benchmarks
+
+### New features
+
+- [ ] SortBy
+- [ ] Geo search
+- [ ] NDJSON
+- [ ] CSV
```
Comment (curquiza) [inline roadmaps/2021.md:50]:
```suggestion
- [ ] CSV
- [ ] Improve analytics
```

File: roadmaps/2021.md [hunk 2]
```diff
@@ -23,9 +23,38 @@ Nothing is definitive: the roadmap can be adapted depending on the need of the u
...
-- [ ] Publish the core-team repositories to crates.io
-- [ ] Release the first RC of MeiliSearch v0.21.0 that integrates the new search engine (Transplant + Milli)
-- [ ] Release v0.21.0 of MeiliSearch
-
+- _Reported, waiting for internal decision_ Publish the core-team repositories to crates.io
+- [X] Release the first RC of MeiliSearch v0.21.0 that integrates the new search engine (Transplant + Milli)
+- _Reported to Q3_ Release v0.21.0 of MeiliSearch
+
+## Q3
+
```
Comment (gmourier) [inline roadmaps/2021.md:33]:
```suggestion
### Goals

- Reduce biggest user's deal-breakers.
- Start to stabilize the API format block by block until v1.0. 
- Understand our user's behaviors.
- Patch v0.21 indexing speed.
```

File: roadmaps/2021.md [hunk 3]
```diff
@@ -23,9 +23,38 @@ Nothing is definitive: the roadmap can be adapted depending on the need of the u
...
+- [ ] SortBy
+- [ ] Geo search
+- [ ] NDJSON
+- [ ] CSV
+- [ ] Improve analytics
+- [ ] Clean API
+  - [ ] Search
+  - [ ] Documents
+  - [ ] Updates
+  - [ ] Indexes
```
Comment (gmourier) [inline roadmaps/2021.md:56]:
```suggestion
- [ ] Indexation benchmarks (Patch v0.21 indexing speed)

### New features

#### Reduce biggest user's deal-breakers
- [ ] SortBy
- [ ] Geo search

#### Patch v0.21 indexing speed
- [ ] Indexing NDJSON file format
- [ ] Indexing CSV file format
- [ ] Indexer optimization

#### Understand our user's behaviors.
- [ ] Branch analytics at feature level (On those planned in this quarter)

#### Stabilize the API format block by block until v1.0.
- [ ] Clean API Format
  - [ ] Search
  - [ ] Documents
  - [ ] Updates
  - [ ] Indexes
```

File: roadmaps/2021.md [hunk 4]
```diff
@@ -23,9 +23,38 @@ Nothing is definitive: the roadmap can be adapted depending on the need of the u
...
+
+### Optimization
+
+- [ ] Indexer
```
Comment (gmourier) [inline roadmaps/2021.md:60]:
```suggestion
```

**Review by gmourier — CHANGES_REQUESTED** [review]
I just reorganized according to the goals of Q3 in order to bring some precisions 😇

File: roadmaps/2021.md (hunk 4 above)
Comment (gmourier) [reply roadmaps/2021.md:60]: I moved the optimization line under the objective of improving indexing performance! Thanks, your work @curquiza!

**Review by gmourier — APPROVED** [review]
LGTM 🥳

**Review by tpayet — APPROVED** [review]
🙌

**Pull request merged**